web/src/saved_snippets_ui.ts

```diff
@@ -108,7 +107,7 @@ function item_button_click_callback(event: JQuery.ClickEvent): void {
     ) {
         confirm_dialog.launch({
             html_heading: $t_html({defaultMessage: "Delete saved snippet?"}),
-            html_body: render_confirm_delete_saved_snippet(),
+            text_subheader: $t({defaultMessage: "This action cannot be undone."}),
```

Comment:
This one forgets to delete `web/templates/confirm_dialog/confirm_delete_saved_snippet.hbs`